src/session/content/message_row/sticker.rs

```diff
@@ -122,7 +91,38 @@ impl MessageSticker {
 
     fn load_sticker(&self, path: &str) {
         let self_ = imp::MessageSticker::from_instance(self);
-        let media = gtk::MediaFile::for_filename(path);
-        self_.picture.set_paintable(Some(&media));
+        let paintable = &self_.paintable;
+
+        let file = gio::File::for_path(path);
+        let future = clone!(@weak paintable => async move {
+            match file.load_bytes_async_future().await {
+                Ok((bytes, _)) => {
+                    let image = webp::Decoder::new(&bytes)
+                        .decode()
+                        .unwrap()
+                        .to_image()
+                        .to_rgba8();
+
+                    let samples = image.into_flat_samples();
+
+                    let (stride, width, height) = samples.extents();
+                    let gtk_stride = stride * width;
+
+                    let bytes = glib::Bytes::from(samples.as_slice());
```

Comment:
It seems that `glib::Bytes::from` is copying the bytes, which is not really great.

Reply:
glib::Bytes::from_owned?

Reply:
If I do that it complains.

```
    |
112 |                     let bytes = glib::Bytes::from_owned(samples.as_slice());
    |                                                         ^^^^^^^^^^^^^^^^^^
    |                                                         |
    |                                                         borrowed value does not live long enough
    |                                                         argument requires that `samples` is borrowed for `'static`
...
121 |                 }
    |                 - `samples` dropped here while still borrowed
```

Reply:
let bytes = glib::Bytes::from_owned(samples.samples);

Reply:
That was it, thanks! :)